genologics/entities.py

```diff
@@ -373,14 +369,13 @@ class BooleanDescriptor(StringDescriptor):
     """An instance attribute containing a boolean value
     represented by an XMl element.
     """
-
     def __get__(self, instance, cls):
-        instance.get()
-        node = self.get_node(instance)
-        if node is None:
-            return None
-        else:
-            return node.text.lower() == 'true'
+        text = super(BooleanDescriptor, self).__get__(instance, cls)
+        if text is not None:
+            return text.lower() == 'true'
+
+    def __set__(self, instance, value):
+        super(BooleanDescriptor, self).__set__(instance, str(value).lower())
```

Comment:
BooleanDescriptor was previously accepting string because it inherited from StringDescriptor.
I was trying to maintain backward compatibility. so bot ```__set__('true')``` and ```__set__(True)``` would work.
It was actually failing on ```__set__(True)``` before: [see test](https://github.com/SciLifeLab/genologics/blob/master/tests/test_entities.py#L140)
Happy to restrict it to the latter if you don't think that's going to cause issue to anybody.